tests/test_filter_ds.py

```diff
@@ -0,0 +1,24 @@
+from FilterDS import FilterDS
+
+def test_initialization():
```

Comment:
This test doesn't do anything.  Is it leftover from the beginning of TDD?  If so, remove it.  If not, add some type of assert (and rename the test, as appropriate).